ArrayUndObject/ÜbungenzuArraysund05.js

```diff
@@ -0,0 +1,37 @@
+const readline = require('readline');
+const rl = readline.createInterface({input: process.stdin, output: process.stdout});
+const prompt = (query) => new Promise((resolve) => rl.question(query, resolve));
+rl.on('close', () => process.exit(0));
+
+async function execute() { // put all your codes here for user inputs
+
+    const dataBase = {
```

Comment:
Hier sollst du ein "Array von Objekten" erstellen, du hast aber ein Objekt erstellt, in dem Arrays sind.

Reply:
*Corrected